packages/standard-validator/__schemas__/arktype.ts

```diff
@@ -26,7 +26,12 @@ const querySortSchema = type({
   order: "'asc'|'desc'",
 })
 
+const headerSchema = type({
+  'User-Agent': 'string',
```

Comment:
Are you intentionally using a `User-Agent` that includes capital letters? Because it contains capital letters, indeed, it will cause a validation error when validating the value.

However, I think this is confusing. So, I think adding a comment or comparing a completely different key would be great.